<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog5</class>
 <widget class="QDialog" name="Dialog5">
  <property name="geometry">
      <rect>
          <x>0</x>
          <y>0</y>
          <width>889</width>
          <height>600</height>
      </rect>
  </property>
     <property name="windowTitle">
         <string>CARİ</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
         <item row="4" column="0">
             <widget class="QLabel" name="label">
                 <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                     </sizepolicy>
                 </property>
                 <property name="font">
                     <font>
                         <pointsize>14</pointsize>
                         <weight>75</weight>
                         <bold>true</bold>
                     </font>
                 </property>
                 <property name="text">
                     <string>Başlama Tarihi</string>
                 </property>
             </widget>
         </item>
         <item row="10" column="0">
             <widget class="QCheckBox" name="checkBox">
                 <property name="text">
                     <string>Bakiyesi olmayanları gösterme</string>
                 </property>
             </widget>
         </item>
         <item row="1" column="0">
             <widget class="QLabel" name="label_3">
                 <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                     </sizepolicy>
                 </property>
                 <property name="font">
                     <font>
                         <pointsize>14</pointsize>
                         <weight>75</weight>
                         <bold>true</bold>
                     </font>
                 </property>
                 <property name="text">
                     <string>Firma Adı</string>
                 </property>
             </widget>
         </item>
         <item row="2" column="0">
             <widget class="QLineEdit" name="lineEdit">
                 <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                     </sizepolicy>
                 </property>
                 <property name="font">
                     <font>
                         <pointsize>14</pointsize>
                     </font>
                 </property>
             </widget>
         </item>
         <item row="5" column="0">
             <widget class="QDateEdit" name="dateEdit">
                 <property name="font">
                     <font>
                         <pointsize>14</pointsize>
                         <weight>75</weight>
                         <bold>true</bold>
                     </font>
                 </property>
                 <property name="calendarPopup">
                     <bool>true</bool>
                 </property>
             </widget>
         </item>
         <item row="6" column="0">
             <widget class="QLabel" name="label_2">
                 <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                     </sizepolicy>
                 </property>
                 <property name="font">
                     <font>
                         <pointsize>14</pointsize>
                         <weight>75</weight>
                         <bold>true</bold>
                     </font>
                 </property>
                 <property name="text">
                     <string>Bitiş Tarihi</string>
                 </property>
             </widget>
         </item>
         <item row="7" column="0">
             <widget class="QDateEdit" name="dateEdit_2">
                 <property name="font">
                     <font>
                         <pointsize>14</pointsize>
                         <weight>75</weight>
                         <bold>true</bold>
                     </font>
                 </property>
                 <property name="calendarPopup">
                     <bool>true</bool>
                 </property>
             </widget>
         </item>
         <item row="9" column="0">
             <widget class="QPushButton" name="pushButton">
                 <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                     </sizepolicy>
                 </property>
                 <property name="font">
                     <font>
                         <pointsize>12</pointsize>
                     </font>
                 </property>
                 <property name="text">
                     <string>Ekstre</string>
                 </property>
             </widget>
         </item>
         <item row="0" column="2" rowspan="16">
             <widget class="QTableWidget" name="tableWidget">
                 <column>
                     <property name="text">
                         <string>.</string>
                     </property>
                 </column>
                 <column>
                     <property name="text">
                         <string>.</string>
                     </property>
                 </column>
                 <column>
                     <property name="text">
                         <string>.</string>
                     </property>
                 </column>
                 <column>
                     <property name="text">
                         <string>.</string>
                     </property>
                 </column>
                 <column>
                     <property name="text">
                         <string>.</string>
                     </property>
                 </column>
                 <column>
                     <property name="text">
                         <string>.</string>
                     </property>
                 </column>
             </widget>
         </item>
         <item row="15" column="0">
             <widget class="QPushButton" name="pushButton_3">
                 <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                     </sizepolicy>
                 </property>
                 <property name="font">
                     <font>
                         <pointsize>12</pointsize>
                     </font>
                 </property>
                 <property name="text">
                     <string>XLS</string>
                 </property>
             </widget>
         </item>
         <item row="14" column="0">
             <widget class="QPushButton" name="pushButton_2">
                 <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                     </sizepolicy>
                 </property>
                 <property name="font">
                     <font>
                         <pointsize>12</pointsize>
                     </font>
                 </property>
                 <property name="text">
                     <string>PDF</string>
                 </property>
             </widget>
         </item>
     </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
